runtime/drivers/bigquery/bigquery.go

```diff
@@ -28,53 +28,22 @@ var spec = drivers.Spec{
 	DocsURL:     "https://docs.rilldata.com/connect/data-source/bigquery",
 	ConfigProperties: []*drivers.PropertySpec{
 		{
-			Key:  "google_application_credentials",
-			Type: drivers.FilePropertyType,
-			Hint: "Enter path of file to load from.",
+			Key:         "google_application_credentials",
+			Type:        drivers.InformationalPropertyType,
+			DisplayName: "GCP credentials",
+			Description: "GCP credentials inferred from your local environment.",
+			Hint:        "Set your local credentials: <code>gcloud auth application-default login</code> Click to learn more.",
+			DocsURL:     "https://docs.rilldata.com/connect/data-source/gcs#rill-developer-local-credentials",
 		},
 		{
 			Key:         "project_id",
 			Type:        drivers.StringPropertyType,
 			Required:    false,
```

Comment:
Did you successfully create a BigQuery connector? I submitted the form without a ProjectID and I hit the below error, which to me implies that we should change `project_id` to required.
<img width="1000" height="910" alt="image" src="https://github.com/user-attachments/assets/1cc6c872-67be-4d3c-ba25-f8f040243860" />

Reply:
Will mark project_id as required

Reply:
bigquery currently extracts projectID from the google application credentials https://github.com/rilldata/rill/blob/main/runtime/drivers/bigquery/bigquery.go#L259

Reply:
Ok I didn't realize it was derived from the Google Application Credentials